WD-Firmware/demo/build/SConstruct

```diff
@@ -154,10 +142,12 @@ print "Building SConscripts files:\n\n-%s" % "\n-".join(objDemo.listSconscripts)
 print strEndHeader
 for strSconscript in objDemo.listSconscripts:
   arrLibs.append(SConscript(strSconscriptPrefix + '%s' % strSconscript, exports='Env'))
-  
-strElfFileName = os.path.join(Env['OUT_DIR_PATH'], strElf_file)
-objBundle = Env.Program(strElfFileName, arrLibs, LIBS=Env['TARGET_LIBS'])
-Env.Size(source=objBundle)
+
+objBundle = Env.Program(Env['ELF_FILE'], arrLibs, LIBS=Env['TARGET_LIBS'])
+# execute post build actions
+AddPostAction(objBundle, Action(utils.fnProduceSectionsSize))

```

Comment:
we might find a better way to do this with depend. DONT change anything! just add a comment for "todo:" to check depend soultion

Reply:
done in 2a82cc9